lib/diagnostic.rb

```diff
@@ -7,36 +7,36 @@ class Diagnostic
   # Assume each entry in the list implements a length method that returns
   # a Fixnum.
   def longest(list)
-    # Your code goes here
+    list.max
```

Comment:
Close but we should pass `max` a block so it knows how to compare the elements:

`list.max { |a, b| a.length <=> b.length }`